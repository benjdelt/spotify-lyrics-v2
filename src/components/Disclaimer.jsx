import { useTranslation } from 'react-i18next';
import '../styles/Disclaimer.css';

function Disclaimer() {

  const { t } = useTranslation();

  return (
    <section className="disclaimer">
      <h2>{ t('disclaimer.header') }</h2>
      <p>
        { t('disclaimer.p1') }
      </p>
      <p>
        { t('disclaimer.p2') }
      </p>
      <p>
          { t('disclaimer.p3') }
      </p>
    </section>
  );
}

export default Disclaimer;
