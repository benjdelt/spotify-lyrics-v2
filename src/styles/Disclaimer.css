.disclaimer {
  width: 40vw;
  margin-top: 60px;
}

@media (max-width: 490px) {
  .disclaimer {
    width: 60vw;
  }
}
