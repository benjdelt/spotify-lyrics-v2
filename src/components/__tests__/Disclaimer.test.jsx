import { render, cleanup } from '@testing-library/react';
import Disclaimer from '../Disclaimer';

afterEach(cleanup)

it('should take a snapshot', () => {
  const { asFragment } = render(<Disclaimer />) 
  expect(asFragment(<Disclaimer />)).toMatchSnapshot()
});
