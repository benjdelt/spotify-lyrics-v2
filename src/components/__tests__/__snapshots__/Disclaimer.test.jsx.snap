// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <section
    class="disclaimer"
  >
    <h2>
      disclaimer.header
    </h2>
    <p>
      disclaimer.p1
    </p>
    <p>
      disclaimer.p2
    </p>
    <p>
      disclaimer.p3
    </p>
  </section>
</DocumentFragment>
`;
